fvm/transactionEnv.go

```diff
@@ -172,7 +187,7 @@ func (e *TransactionEnv) GetAuthorizedAccountsForContractUpdates() []common.Addr
 		e.ctx.Logger.Warn().Msg("failed to read contract deployment authorized accounts from service account. using default behaviour instead.")
 		return defaultAccounts
 	}
-	addresses, ok := utils.OptionalCadenceValueToAddressSlice(value)
+	addresses, ok := utils.CadenceValueToAddressSlice(value)
```

Comment:
@turbolent ReadStored now returns T instead of optional of T, is this expected?

Reply:
Good catch! Yes, the behaviour changed, instead of `OptionalValue` with `NilValue`/`SomeValue`, the function now just returns `nil` and the value as-is if it exists. 

I forgot to update the comments during the refactor, I fixed it in https://github.com/onflow/cadence/pull/1439